common/src/test/java/com/lightstep/tracer/shared/OptionsTest.java

```diff
@@ -102,6 +109,14 @@ public void testOptionsBuilder_httpsNoPortProvided() throws Exception {
         assertEquals(DEFAULT_SECURE_PORT, options.collectorUrl.getPort());
     }
 
+    @Test
+    public void testOptionsBuilder_defaultScopeManager() throws Exception {
+        Options options = new Options.OptionsBuilder()
+                .build();
+
+        assertEquals(ThreadLocalScopeManager.class, options.scopeManager.getClass());
+    }
```

Comment:
I'd personally change it to `assertTrue(options.scopeManager() instanceof ThreadLocalScopeManager)`, as it is more readable.